app/views/users/_admin_grant_permissions.html.erb

```diff
@@ -0,0 +1,72 @@
+<% namesHash = name_and_text %>
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+
+      <div class="modal-header">
+        <button type="button" class="close" 
+                data-dismiss="modal" 
+                aria-label="Close">
+                <span aria-hidden="true">&times;</span></button>
+        <h1 class="modal-title"><%= _('Edit User Privileges') %></h1>
+      </div>
+      
+      <%= form_tag( admin_update_permissions_user_path(@user), method: :put) do %>
+        <div class="modal-body" style="text-align: left">
+          <p><%= @user.name(false) %><%= _(' current user privileges are: ') %></p>
+
+          <ul class="list-group">
+            <% if current_user.can_super_admin? %>
```

Comment:
According to the snapshot from #794 first are listed org_admin privileges, so please swap the order